### ISSUE TITLE
ACCoeff and NLTE structures missing (?) from SpcCoeff_netCDF_IO.f90 

### ISSUE THREAD

**Issue by BenjaminTJohnson** [JCSDA/CRTMv3#101]
In SpcCoeff_Binary_IO.f90, there are structures to handle ACCoeff/NLTE structures.   e.g.:
```
  USE ACCoeff_Define     , ONLY: ACCoeff_Associated
  USE ACCoeff_Binary_IO  , ONLY: ACCoeff_Binary_ReadFile , &
                                 ACCoeff_Binary_WriteFile
  USE NLTECoeff_Define   , ONLY: NLTECoeff_Associated
  USE NLTECoeff_Binary_IO, ONLY: NLTECoeff_Binary_ReadFile , &
                                 NLTECoeff_Binary_WriteFile
<...>
   IF ( ac_present == DATA_PRESENT ) THEN
      err_stat = ACCoeff_Binary_ReadFile( &
                   Filename         , &
                   SpcCoeff%AC      , &
                   No_Close = .TRUE., &
                   Quiet    = Quiet , &
                   Debug    = Debug   )
      IF ( err_stat /= SUCCESS ) THEN
        msg = 'Error reading antenna correction data.'
        CALL Read_Cleanup(); RETURN
      END IF
      ! ..Check that the ACCoeff data is for the same sensor                                                                                                                                          
      IF ( SpcCoeff%Sensor_Id           /= SpcCoeff%AC%Sensor_Id        .OR. &
           SpcCoeff%WMO_Satellite_Id    /= SpcCoeff%AC%WMO_Satellite_Id .OR. &
           SpcCoeff%WMO_Sensor_Id       /= SpcCoeff%AC%WMO_Sensor_Id    .OR. &
           ANY( SpcCoeff%Sensor_Channel /= SpcCoeff%AC%Sensor_Channel ) ) THEN
        msg = 'Antenna correction sensor information is inconsistent with SpcCoeff'
        CALL Read_CleanUp(); RETURN
      END IF
    END IF
<...>
```

conversion from binary to netcdf, therefore appears to ignore these structures, which is problematic.   (e.g., converting amsua_metop-c_v2.SpcCoeff.bin to netCDF does not produce the ACCoeff information in the netCDF file.)

Am I missing something?

**Comment by YingtaoMa** [comment]
HI Ben,
Could you forward me the code you used to convert the binary SpcCoeff to
netCDF format?

Thanks

**Comment by BenjaminTJohnson** [comment]
@YingtaoMa  I was using one that I wrote myself based on SpcCoeff_NC2BIN (just reversing the various things)  -- but I figured out a better solution for the moment, but it's not great.  

(1) Take the binary file that has ACCoeff information in it (e.g., `amsua_metop-c_v2.SpcCoeff.bin`), and convert it to netCDF using existing BIN2NC routines.
(2) recompile all of the code in LE mode (or BE, whatever the opposite of the binary file is). 
(3) Then run NC2BIN and have the appropriate amsua_metop-c.ACCoeff.nc file present in the same directory, and it will grab the additional AC stuff and create a BE or LE file that has the ACCoeff information in it.   This assumes that you have the various ACCoeff files.

A better solution would be for the SpcCoeff_netCDF_IO routines to properly read/write the NLTECoeff and ACCoeff structures when present in the internal SpcCoeff structure.    As far as I can tell, the existing SpcCoeff_netCDF_IO routines are all missing the ACCoeff and NLTECoeff read/write capabilities.
